src/nanoemoji/maximum_color.py

```diff
@@ -272,9 +376,15 @@ def _run(argv):
             _write_preamble(nw, input_font)
 
             if color_table == "COLR":
-                _generate_svg_from_colr(nw, input_font, font)
+                color_font, picosvg_files = _generate_svg_from_colr(
+                    nw, input_font, font
+                )
             else:
-                _generate_colr_from_svg(nw, input_font, font)
+                color_font, picosvg_files = _generate_colr_from_svg(
+                    nw, input_font, font
+                )
+
+            _generate_cbdt(nw, input_font, font, color_font, picosvg_files)
```

Comment:
Why to make a CBDT table from it do we need to run picosvg on the svg extracts? Can't we use the svg as they are. Resvg is definitely capable to rasterise them. I don't see the reason to do that. It just adds an extra layer of potential bugs. The two `_generate_*` methods above should just return the extracted SVGs (or converted from COLR) without (or before) running picosvg on them.

Reply:
Because I'd like the generated tables to be consistent. For an ot-svg input I think odds are nanoemoji would do a better job (since most existing ones seem to do svg per glyph with no sharing) so arguably we should rebuild the svg table in that situation.

Resolving but plz reopen if that argument is absurd.